packages/java-jaxrs-common/templates/wrapperNested.hbs

```diff
@@ -1,6 +1,6 @@
 {{>frag/pojoDocumentation}}
 {{>frag/wrapperHeader}}
-public static class {{className name}} {{>pojoImplements}}{
+class {{className name}} {{>pojoImplements}}{
```

Comment:
I suspect that we might need to know whether we're outputting in an interface or in a class before making this distinction.

Reply:
I have removed it from this PR. I will look at how to do it in another one later.